docs/contributing/Compiler Test Plan.md

```diff
@@ -72,33 +74,37 @@ This document provides guidance for thinking about language interactions and tes
     - Ref / out parameters
     - Compound operators (+=, /=, etc ..) 
     - Assignment exprs
+- Ref returns
 
- 
 # Misc
 - reserved keywords (sometimes contextual)
 - pre-processing directives
 - COM interop
 - modopt and modreq
 - ref assemblies
 
-
 # Testing in interaction with other components
 Interaction with IDE, Debugger, and EnC should be worked out with relevant teams. A few highlights:
 - IDE
```

Comment:
We should consider just linking to the IDE test plan and keeping it as a separate artifact.